api/v1beta1/awsmachine_types.go

```diff
@@ -157,6 +157,10 @@ type AWSMachineSpec struct {
 	// +optional
 	// +kubebuilder:validation:Enum:=default;dedicated;host
 	Tenancy string `json:"tenancy,omitempty"`
+
+	// CapacityReservationID specifies the capacity reservation resource id that should be
```

Comment:
I think you are missing the rest of the sentence :)

Reply:
fixed